test/functional/interface_zmq.py

```diff
@@ -10,12 +10,11 @@
 from test_framework.messages import CTransaction
 from test_framework.util import (
     assert_equal,
+    connect_nodes,
     hash256,
 )
 from io import BytesIO
 
-ADDRESS = "tcp://127.0.0.1:28332"
```

Comment:
in commit be08278a3f7348a346b60421b2c929a86020a4db

Would be nice to give a rationale for this "refactor". Otherwise, I'd be NACK on this commit

Reply:
I think this was moved into the specific test because the other test uses a different address, so there's no reason to share this globally

Reply:
Yes, the refactor is to avoid state and constants across test cases.

Reply:
Avoiding constants across test cases is not a goal in itself though

Reply:
I mean that each test case needs a different address and I thought having a constant for each test is pointless, that's why I moved the value inline.

But I'll happily revert, the refactor is just to simplify the next commit - I'll add a rationale.

Reply:
Sorry, didn't realize they were different addresses. ACK in that case.

Reply:
Rationale added.